### PR TITLE
Complete Portfolio - Mobile Version

### PR DESCRIPTION
**Pull request by misheck12**
In this project, I upgraded the skeleton portfolio version with the following sections: 
> Add Works section
> Add about myself 
>styled the html and css sections

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,36 +6,176 @@
...
           <li><a href="#"><img src="./images/angellist.png" alt="social"></a></li>
         </ul>
       </div>
-    </section>
\ No newline at end of file
+    </section>
+    <section class="work-section">
+      <div class="project-card">
+
+        <img src="images/photo1.png" alt="First Card">
```
Comment (prolajumokeoni) [inline index.html:43]: ## Optional
- [x] You can add this  image as a background image using [CSS and not img tag ](https://www.w3schools.com/cssref/pr_background-image.asp) 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @misheck12 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there 🥇 
## Highlights
- Good PR message ✔️ 
- No linter errors ✔️ 
- Good readme ✔️ 

## Required Changes ♻️

- Kindly add a more descriptive short pull request title describing what changes you made in your pull request 👍🏿 
**_Check inline comments_**
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
.
You can also consider:
- Kindly commit your code in smaller chunks with meaningful commit messages, this helps you understand your code better and changes made at certain intervals in case you want to revert changes 🤝 

Almost-gif-simpson



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -6,36 +6,176 @@
...
+          <p> A daily selection of privtely personalized reads; no accounts or sign-ups required. </p>
+          <div class="language1 inline">
+            <ul>
+              <li class="langu">html</li>
+              <li class="langu">css</li>
+              <li class="langu">javascript</li>
+            </ul>
+          </div>
+        </div>
+        <button type="button" class="btn-one">See project</button>
```
Comment (prolajumokeoni) [inline index.html:84]: Figma Design   |  Your Design
:-------------------------:|:-------------------------:
image  |  hisbtn

- [x] Kindly adjust padding for li's in your design 
- [x] Kindly make border for buttons like the figma design
### These changes will make your design look exactly like the figma design, kindly do this for all project card  li's and  buttons  💪🏿

File: index.html [hunk 3]
```diff
@@ -6,36 +6,176 @@
...
```
Comment (prolajumokeoni) [inline index.html:126]: Figma Design   |  Your Design
:-------------------------:|:-------------------------:
image  |  hisbtn

- [x] Kindly adjust padding for li's in your design 
- [x] Kindly make border for buttons like the figma design
### These changes will make your design look exactly like the figma design, kindly do this for all project card  li's and  buttons  💪🏿

File: index.html [hunk 4]
```diff
@@ -6,36 +6,176 @@
...
```
Comment (prolajumokeoni) [inline index.html:106]: Figma Design   |  Your Design
:-------------------------:|:-------------------------:
image  |  hisbtn

- [x] Kindly adjust padding for li's in your design 
- [x] Kindly make border for buttons like the figma design
### These changes will make your design look exactly like the figma design, kindly do this for all project card  li's and  buttons  💪🏿

File: index.html [hunk 5]
```diff
@@ -6,36 +6,176 @@
...
+          <p> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
+          <div class="language1 inline">
+            <ul>
+              <li class="langu">html</li>
+              <li class="langu">css</li>
+              <li class="langu">javascript</li>
+            </ul>
+          </div>
+        </div>
+        <button type="button" class="btn-one">See project</button>
```
Comment (prolajumokeoni) [inline index.html:62]: Figma Design   |  Your Design
:-------------------------:|:-------------------------:
image  |  hisbtn

- [x] Kindly adjust padding for li's in your design 
- [x] Kindly make border for buttons like the figma design
### These changes will make your design look exactly like the figma design, kindly do this for all project card  li's and  buttons  💪🏿

File: index.html (hunk 1 above)
Comment (misheck12) [reply index.html:43]: Hi @prolajumokeoni ,

Thank you for your review. However, don't you think having this in an image tag is required, looking at the nature of its content, the image tag might be useful for SEO. Kindly advise ASAP.

> HappyCoding :woman_technologist:

Comment (prolajumokeoni) [reply index.html:43]: > Hi @prolajumokeoni ,
> 
> Thank you for your review. However, don't you think having this in an image tag is required, looking at the nature of its content, the image tag might be useful for SEO. Kindly advise ASAP.
> 
> > HappyCoding 👩‍💻

Hi @misheck12 you can add it the way you prefer, in the figma design it was added as a background image, kindly check the figma design and great point by the way🤗

File: index.html (hunk 5 above)
Comment (misheck12) [reply index.html:62]: Actioned.

File: index.html (hunk 4 above)
Comment (misheck12) [reply index.html:106]: Actioned :smiley: .

Thanks a billion for that precise eye on the design.

File: index.html (hunk 1 above)
Comment (prolajumokeoni) [reply index.html:43]: @misheck12 don't forget to change your pull request title to a more descriptive title that give information about your pull request, it's a required change 👍

File: index.html (hunk 2 above)
Comment (misheck12) [reply index.html:84]: Actioned.

File: index.html (hunk 3 above)
Comment (misheck12) [reply index.html:126]: <Actioned/>

File: index.html (hunk 1 above)
Comment (misheck12) [reply index.html:43]: Thanks a lot @prolajumokeoni. I appreciate your professional advise.
I have made the other changes, and I'll leave the image as is, following the aforementioned SEO impact.

> HappyCoding :hugs:

File: index.html (hunk 5 above)
Comment (Donard97) [inline index.html:62]: - [x] You did a great job implementing required changes from the other reviewer but you have a few more things to do. Kindly arrange list on a line and make button smaller following [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).
Here you have a screenshot of how I am seeing it:
image

File: index.html (hunk 2 above)
Comment (Donard97) [inline index.html:84]: - [x] Kindly fix the card content style following [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). Keep rocking 💪

File: index.html (hunk 4 above)
Comment (Donard97) [inline index.html:106]: - [x] Kindly fix the card content style following [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). Keep rocking 💪

File: index.html (hunk 3 above)
Comment (Donard97) [inline index.html:126]: - [x] Kindly fix the card content style following [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). Keep rocking 💪

File: index.html (hunk 1 above)
Comment (prolajumokeoni) [reply index.html:43]: Nice work @misheck12

**Review by Donard97 — CHANGES_REQUESTED** [review]
## Hello @misheck12 , 👋  
    
Great work on making the changes requested by a previous reviewer 👏 💯 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
    
giphy (5)


## Required Changes ♻️

- [x] Please check the comments under the review.
    
## Optional suggestions
    
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
    
**Cheers and Happy coding!👏👏👏**
    
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
    
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
    
------
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 5 above)
Comment (misheck12) [reply index.html:62]: Actioned. > Thanks a lot for your review.

File: index.html (hunk 3 above)
Comment (misheck12) [reply index.html:126]: Hello @Donard97, Thank you for your awesome review :-) . I have implemented the changes. 

Fingers crossed, hoping and praying that next reviewer approves my project.

> HappyCoding :man_technologist:

**Review by Wes-Isaac — APPROVED** [review]
Hi @misheck12 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**